### PR TITLE
[Audio] Fix PCM format and use PipedAudioStream in sources

### PR DESCRIPTION
**Pull request by GiviMAD**
Hello,
I want to merge these changes related to the core PR https://github.com/openhab/openhab-core/pull/3960.

It includes changes on the Speech-to-text services to advertise compatibility and handle raw PCM streams (no container header).
Also updates de pulseaudio source to use the new PipedStream class and fixes its format which should use container CONTAINER_NONE (my bad).

NOTE: I have removed the opus support from the googlestt because it's unused.

### CODE REVIEW

**Comment by GiviMAD** [comment]
Required PR has been merged last weekend.

File: bundles/org.openhab.binding.pulseaudio/src/main/java/org/openhab/binding/pulseaudio/internal/PulseAudioAudioSource.java [hunk 1]
```diff
@@ -76,27 +70,22 @@ public AudioStream getInputStream(AudioFormat audioFormat) throws AudioException
...
-                        @Override
-                        public void close() throws IOException {
-                            unregisterPipe(pipeOutput);
-                            super.close();
+                    var audioStream = streamGroup.getAudioStreamInGroup();
+                    audioStream.onClose(() -> {
+                        minusClientCount();
+                        try {
+                            Thread.sleep(0);
+                        } catch (InterruptedException ignored) {
```
Comment (lolodomo) [inline bundles/org.openhab.binding.pulseaudio/src/main/java/org/openhab/binding/pulseaudio/internal/PulseAudioAudioSource.java:81]: What is the purpose of this sleep with 0 delay ?

**Review by lolodomo — COMMENTED** [review]
Just one question

**Comment by lolodomo** [comment]
And a second question: you removed the support of OGG or MP3 audio formats. Is it because they were in fact not really supported ?

**Comment by lolodomo** [comment]
Please note that the build failed.

File: bundles/org.openhab.binding.pulseaudio/src/main/java/org/openhab/binding/pulseaudio/internal/PulseAudioAudioSource.java (hunk 1 above)
Comment (GiviMAD) [reply bundles/org.openhab.binding.pulseaudio/src/main/java/org/openhab/binding/pulseaudio/internal/PulseAudioAudioSource.java]: It should be there before refactoring it, I have checked with the latest snapshot and it works correctly without it.

**Comment by GiviMAD** [comment]
> And a second question: you removed the support of OGG or MP3 audio formats. Is it because they were in fact not really supported ?

The [doc](https://cloud.google.com/speech-to-text/docs/reference/rpc/google.cloud.speech.v1#audioencoding) says OGG is supported, but as there are no sources available using that format I never tested it, so I though it was better to remove it to avoid confusion.

**Review by lolodomo — APPROVED** [review]
LGTM, thank you

**Pull request merged**